database/migrations/2024_06_25_073625_uuid_extension.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\DB;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        DB::statement('CREATE EXTENSION IF NOT EXISTS "uuid-ossp";');
+    }
+
+    /**
+     * Reverse the migrations.
+     */
+    public function down(): void
+    {
+        DB::statement('DROP EXTENSION IF EXISTS "uuid-ossp";');
+    }
+};
```

Comment:
this file should be the first one when running the migrations so by changing the date of the file it make it the first one so  it executes first